src/DynamicRupture/FrictionLaws/FrictionSolverCommon.h

```diff
@@ -136,6 +140,33 @@ inline void precomputeStressFromQInterpolated(
                   qIMinus[o][T2][i] * invZsNeig);
     }
   }
+#else
+  seissol::dynamicRupture::kernel::computeTheta krnl;
```

Comment:
We could, but I don't see a benefit. The impedance matrix is diagonal in the elastic case. Also this would break the GPU implementation of this method.